pkg/minishift/oc/oc.go

```diff
@@ -94,16 +94,15 @@ func (oc *OcRunner) AddSudoerRoleForUser(user string) error {
 func (oc *OcRunner) AddSystemAdminEntryToKubeConfig(ocPath string) error {
 	var existingKubeConfig, newKubeConfig *clientcmdapi.Config
 
-	user, err := user.Current()
+	minishiftKubeConfigPath := oc.KubeConfigPath
+	kubeConfigGlobalPath, err := getKubeConfigGlobalPath()
```

Comment:
globalKubeConfigPath sounds better as a variable name here.

Reply:
Same with function name too i.e. `getGlobalKubeConfigPath`

Reply:
@LalatenduMohanty it was there before so I didn't change it but can be updated.